encrypted-media-respec.html

```diff
@@ -1998,11 +1989,12 @@ <h5>
                   </li>
                   <li>
                     <p>
-                      If <var>content type</var> is not strictly a <var>audio/video type</var>,
+                      If <var>mimeType</var> is not strictly an <var>audio/video type</var>,
                       continue to the next iteration.
                     </p>
                     <p class="note">
-                      For example, if <var>audio/video type</var> is Video and the top-level type
+                      For example, if <var>audio/video type</var> is Video and <var>mimeType</var>'s
+                      <var>type</var>
```

Comment:
It seems weird to use `<var>` to refer to a member. Can't you use `[=MIME type/type=]` or some such syntax? Or maybe `<a for="MIME type">type</a>`?

Reply:
Done, thanks.